Dockerfile

```diff
@@ -0,0 +1,47 @@
+FROM ubuntu:latest
+
+# Setup environment variables for Docker image
+ENV FC=/usr/bin/mpif90 CC=/usr/bin/mpicc CXX=/usr/bin/mpicxx \
+    PATH=/opt/openmc/bin:/opt/conda/bin:/opt/NJOY2016/build:$PATH \
+    LD_LIBRARY_PATH=/opt/openmc/lib:$LD_LIBRARY_PATH \
+    OPENMC_CROSS_SECTIONS=/opt/openmc/data/nndc_hdf5/cross_sections.xml \
+    OPENMC_MULTIPOLE_LIBRARY=/opt/openmc/data/WMP_Library \
+    OPENMC_ENDF_DATA=/opt/openmc/data/endf-b-vii.1
+
+# Install dependencies from Debian package manager
+RUN apt-get update -y && \
+    apt-get upgrade -y && \
+    apt-get install -y wget git emacs && \
+    apt-get install -y gfortran g++ cmake && \
+    apt-get install -y mpich libmpich-dev && \
+    apt-get install -y libhdf5-serial-dev libhdf5-mpich-dev && \
+    apt-get autoremove
+
+# Download Miniconda3 and install Python dependencies
+RUN wget --quiet https://repo.continuum.io/miniconda/Miniconda3-latest-Linux-x86_64.sh -O ~/miniconda.sh && \
```

Comment:
Fine by me - I just chose conda since I wanted to have it installed on the image I'll be doing development from, but I can install it separately.

Reply:
Ah, you're actually using it :) My primary concern was just regarding the size of the image. Do you know if it will make a difference whether we use conda or the system-provided Python?

Reply:
First, I like the idea of using the system-provided Python to simplify the `Dockerfile` and to more closely mirror the installation instructions. As for image size, about 240MB is saved by dropping Miniconda: the Miniconda image is 5.21GB while the system image is 4.97GB (2.5GB consumed by the nuclear data). I'll probably continue to use Miniconda in my own personal image for development, but fortunately Docker makes it simple to manage multiple images :-)